python/samples/getting_started/evaluation/azure_ai_foundry/evaluation/README.md

```diff
@@ -0,0 +1,75 @@
+# Self-Reflection Evaluation Sample
+
+This sample demonstrates the self-reflection pattern using Agent Framework and Azure AI Foundry's Groundedness Evaluator. For details, see [Reflexion: Language Agents with Verbal Reinforcement Learning](https://arxiv.org/abs/2303.11366) (NeurIPS 2023).
+
+## Overview
+
+**What it demonstrates:**
+- Iterative self-reflection loop that automatically improves responses based on groundedness evaluation
+- Batch processing of prompts from Parquet files with progress tracking
+- Using `AzureOpenAIChatClient` with Azure CLI authentication
+- Comprehensive summary statistics and detailed result tracking
+
+## Prerequisites
+
+### Azure Resources
+- **Azure OpenAI**: Deploy models (default: gpt-4.1 for both agent and judge)
+- **Azure CLI**: Run `az login` to authenticate
+
+### Python Environment
+```bash
+pip install agent-framework azure-ai-evaluation azure-identity pandas python-dotenv
```

Comment:
azure-identity is a dependency of AF, so not needed, and I would only call for installing core, since that covers the use case:

```suggestion
pip install agent-framework-core azure-ai-evaluation pandas --pre
```